src/OrbitQt/MoveFilesDialog.cpp

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "OrbitQt/MoveFilesDialog.h"
+
+#include <QLabel>
+#include <QPlainTextEdit>
+#include <QScrollBar>
+
+#include "ui_MoveFilesDialog.h"
+
+namespace orbit_qt {
+
+namespace {
+
+class MoveFilesDialogImpl : public MoveFilesDialog {
```

Comment:
Why an Impl class?

Reply:
well.. It started when I decided to introduce include folder to OrbitQt, once I did that CMake operating under assumption that .ui file must me in the same folder as file that includes it, refused to compile the header. I have decided to move include to .cpp file which led to this (by Impl we avoid mentioning Ui::MoveFilesDialog in the header file).

Reply:
Ah, I see. In Qt you usually forward declare `Ui::MoveFilesDialog` in the header and only include `ui_MoveFilesDialog.cpp` in the cpp file. For that to work you can't have a compiler-generated destructor. You have to define it explicitly in the cpp file, for example by defaulting it: `MoveFilesDialog::~MoveFilesDialog() = default;`.

But I guess it doesn't make a huge difference. I personally would prefer the solution without Impl, but feel free to ignore.